mmv1/products/compute/terraform.yaml

```diff
@@ -2845,6 +2845,37 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_value: :TLS_1_0
       warnings: !ruby/object:Overrides::Terraform::PropertyOverride
         exclude: true
+  # Find that Region SSL Policy needs the same custom code as SSL Policy
+  RegionSslPolicy: !ruby/object:Overrides::Terraform::ResourceOverride
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      constants:  'templates/terraform/constants/region_ssl_policy.erb'
+      resource_definition: 'templates/terraform/resource_definition/region_ssl_policy.erb'
+      #update_encoder: 'templates/terraform/update_encoder/region_ssl_policy.erb'
```

Comment:
I think we can probably fully remove this then, correct?

Reply:
The `update_encoder` can be full removed. The `resource_definition` and `constants` are managing the dependency between the `profile` and `customFeatures` fields from what I can tell.